examples/src/main/resources/getGpuResources.sh

```diff
@@ -0,0 +1,2 @@
++# This script uses nvidia-smi command so only works with nvidia GPUs. This is a very basic script that lists the index's of the available GPU's where the executor was allocated.
```

Comment:
add license header.

Also it might be good if we reference this script from some of the discoveryScript documentation

Reply:
do cluster managers do something special so that with multiple executors, a script like this finds the right subset of gpus?  I assume standalone does not, but maybe others do?  That is worth mentioning somewhere as well (its also fine if that part of the story is in a later jira).

Reply:
the script as is will find all GPU's visible to it, so if the cluster manager doesn't isolate it will find all on the host.  Standalone mode, I would expect to use the --gpuDevices parameter to the Executor.    It would be good to add a warning in the config description and here about that, or say the config isnt' supported in standalone mode.